### PR TITLE
Formatting and Style

### PR DESCRIPTION
**Pull request by shoenig**
- separate imports by origin
- add a few documentation comments
- add binary to .gitignore

### CODE REVIEW

**Comment by samcf** [comment]
Thanks for the PR shoenig. I'm afraid I did a bit of work before I even noticed this here.

I find the daemon package a little wonky for my tastes - what's the rationale behind this? It seems like a bunch of code was just pushed around. I admit the main function is a little long.

Configurable ports looks pretty neat, I am definitely digging that. I'll have to take a look at your package (hehe).

Since I am absolutely trash at git, I'm just going to steal the parts I like (formatting, testing, config) and reject the PR so I get all the credit. Thanks for the PR again though, I expect more like it and I'll make sure to not get too far ahead so I can merge them more easily.

**Comment by saml** [comment]
:+1:

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.exe
 hnefatafl.db
-hnefatafl.log
\ No newline at end of file
+hnefatafl.log
```
Comment (saml) [inline .gitignore]: *.log

File: daemon/daemon.go [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+	_ "github.com/mxk/go-sqlite/sqlite3"
+	"github.com/samcf/hnefatafld/tafl"
+	hash "github.com/speps/go-hashids"
+)
+
+const (
+	DSN = "hnefatafl.db"
+)
+
+type Daemon struct {
```
Comment (saml) [inline daemon/daemon.go]: Public type should have documentation.

File: daemon/daemon.go [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+	db       *sqlx.DB
+	hub      Hub
+	sideMap  map[string]tafl.Side
+	upgrader websocket.Upgrader
+	port     int
+	wsPort   int
+}
+
+func NewDaemon(port, wsPort int) *Daemon {
```
Comment (saml) [inline daemon/daemon.go]: Please document public functions. `golint` catches most.

**Pull request closed without merging**